exchange/src/main/java/org/geneontology/gocam/exchange/BioPaxtoGO.java

```diff
@@ -181,54 +182,18 @@ void convert(
 		}
 		//will determine how classes for physical entities are handled
 		if(entityStrategy.equals(EntityStrategy.YeastCyc)) {
-			//neo needs e.g.
-			//http://identifiers.org/sgd/S000001024
-			//we get e.g.
-			//YeastCyc YIL155C-MONOMER
-			//we get the mapping from a YeastCyc GPI file downloaded from http://sgd-archive.yeastgenome.org/curation/literature/ 
-			//e.g. http://sgd-archive.yeastgenome.org/curation/literature/gp_information.559292_sgd.gpi.gz 
-			BufferedReader reader = new BufferedReader(new FileReader("./target/classes/YeastCyc/gp_information.559292_sgd"));
-			String line = reader.readLine();
-			while(line!=null) {
-				if(line.startsWith("!")) {
-					line = reader.readLine();
-				}else {
-					String[] cols = line.split("	");
-					String yeastcyc = cols[4];
-					Set<String> yeastcyc_ids = new HashSet<String>();
-					if(yeastcyc.contains("|")) {
-						String[] ids = yeastcyc.split("\\|");
-						for(String id : ids) {
-							yeastcyc_ids.add(id);
-						}
-					}else {
-						yeastcyc_ids.add(yeastcyc);
-					}
-					if(cols[0].contentEquals("ComplexPortal")) {
-						for(String yeastacc : yeastcyc_ids) {
-							accession_neo.put(yeastacc, "http://purl.obolibrary.org/obo/ComplexPortal_"+cols[1]);
-						}	
-					}else if(cols.length>=8) {
-						String sgd = cols[8];
-						Set<String> sgd_ids = new HashSet<String>();
-						if(sgd.contains("|")) {
-							String[] ids = sgd.split("\\|");
-							for(String id : ids) {
-								sgd_ids.add(id);
-							}
-						}else {
-							sgd_ids.add(sgd);
-						}				
-						for(String yeastacc : yeastcyc_ids) {
-							for(String sgdid : sgd_ids) {
-								accession_neo.put(yeastacc, sgdid.replace("SGD:", "http://identifiers.org/sgd/"));
-							}
-						}					
-					}
-					line = reader.readLine();
-				}
+			String sgdGPIPath = "./target/classes/YeastCyc/gp_information.559292_sgd";
```

Comment:
It would be better to access this is a class path resource, so that it works when this is packaged into a jar. E.g., https://github.com/geneontology/minerva/blob/daa78af32e316dab534fdc64ba3a2dbfa5f954ad/minerva-core/src/main/java/org/geneontology/minerva/curie/DefaultCurieHandler.java#L62

Reply:
Fixed in e55ddd6